install-edm-windows.cmd

```diff
@@ -9,7 +9,7 @@ FOR /F "tokens=1,2,3 delims=." %%a in ("%INSTALL_EDM_VERSION%") do (
 )
 
 SET EDM_MAJOR_MINOR=%MAJOR%.%MINOR%
-SET EDM_PACKAGE=edm_%INSTALL_EDM_VERSION%_x86_64.msi
+SET EDM_PACKAGE=edm_cli_%INSTALL_EDM_VERSION%_x86_64.msi
```

Comment:
You'll need to replace `TARGETDIR` with `EDMAPPDIR`. You're already using `C:/Enthought/edm` as the edm path in the appveyor config file so that's what you need to be using for `EDMAPPDIR`.

Reply:
Thanks